### DISCUSSION TITLE
Can't get this to work on 128b9

### DISCUSSION THREAD

**Discussion by neilio** [artsyfriedchicken/EdgyArc-fr#57] [Q&A]
As the subject says - I double-checked the install and made sure everything was copied over correctly and set in about:config, but the address bar doesn't change, nor does the tab bar or forward button (should be hidden).

Is this an incompatibility with 128b9? I'm using the b11 edgyarc-fr release

**Comment by artsyfriedchicken** [comment]
I am not entirely sure why. its working on mine. are you SURE you went through all the steps?? 
![Screenshot 2024-06-30 at 10 14 00 AM](https://github.com/artsyfriedchicken/EdgyArc-fr/assets/100123017/da294589-f912-442b-93db-6136e34547bb)

**Comment by artsyfriedchicken** [comment]
Please make sure you have the following enabled/initialised as booleans 


uc.tweak.hide-tabs-bar 
uc.tweak.hide-forward-button 
uc.tweak.rounded-corners 
uc.tweak.floating-tabs 


toolkit.legacyUserProfileCustomizations.stylesheets
svg.context-properties.content.enabled
layout.css.color-mix.enabled
layout.css.light-dark.enabled
layout.css.has-selector.enabled
widget.macos.titlebar-blend-mode.behind-window

**Reply by neilio** [reply]
Yes, I have all of those set to true. It's the oddest thing - it works perfectly on my macs, but not on my Windows laptop and I followed the same steps.
